jyulb/fileio/isothermal/jyu_engine.py

```diff
@@ -49,10 +52,11 @@ def __init__(self):
         """Initialize and set default parameters for CM, BC, SP, and SD."""
         # Definition of CM, SP, BC, and SD data components
         self._data = {}
-        self._lattice = None
+        self._proxy_lattice = None
         self.CM = {}
         self.SP = {}
-        self.BC = DataContainer()
+        self.SD = {}
```

Comment:
Just a note that we should, at some point, change CM, SP and BC dictionaries back to DataContainers, but it requires that DataContainer supports extensions to CUBA class (or that we have completed building an extensive CUBA list that covers all the use-cases in SimPhoNy).